.github/workflows/test-documentation-build.yml

```diff
@@ -0,0 +1,31 @@
+name: Test Traits documentation build
+
+on:
+  pull_request
+
+jobs:
+  docs:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.6, 3.8]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies and local packages
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        python -m pip install .[docs]
+    - name: Create clean test directory
+      run: |
+        mkdir testdir
```

Comment:
We don't need a clean test directory for the docs sphinx, do we?

Reply:
Ah, I failed to cover all my copy-and-paste tracks. No, we don't.